examples/interoperability/webpack.config.js

```diff
@@ -0,0 +1,31 @@
+const {resolve} = require('path');
+const webpack = require('webpack');
+
+module.exports = {
+  entry: {
+    app: resolve('./app.js')
+  },
+  devtool: 'source-maps',
+  module: {
+    rules: [{
+      test: /\.js$/,
+      loader: 'buble-loader',
+      include: [resolve('.')],
+      exclude: [/node_modules/],
+      options: {
+        objectAssign: 'Object.assign'
+      }
+    }, {
+      // Needed to inline deck.gl GLSL shaders
+      include: [resolve(__dirname, '../../src')],
```

Comment:
still needed until I finish the shaders in the example.
will cut in the next PR.